Update items.json to comply with Minecraft 1.13.

HyKurtis opened the pull request:
Added new 1.13 items to the items.json file with the appropriate aliases.

Added new text file with a list of all the new items added to items.json as a form of checklist in case I forgot to add any items.

HyKurtis committed:
[UNTESTED] Added new 1.13 items to items.json with appropriate aliases.

HyKurtis committed:
Re-fixed grass alias after it was overwritten in the last commit.

dscalzi commented:
I will review this now 👍

dscalzi commented:
I will update this as I go along.

### Blocks
- [x] BLUE_ICE
- [x] PUMPKIN
- [x] TROPICAL_FISH_BUCKET
- [x] PUFFERFISH_BUCKET
- [x] COD_BUCKET
- [x] SALMON_BUCKET
- [x] DRIED_KELP
- [x] TURTLE_HELMET
- [x] TRIDENT
- [x] KELP
- [x] DRIED_KELP_BLOCK
- [x] TURTLE_EGG
- [x] HEART_OF_THE_SEA
- [x] BIRCH_PRESSURE_PLATE
- [x] SPRUCE_PRESSRE_PLATE
- [x] JUNGLE_PRESSURE_PLATE
- [x] ACACIA_PRESSURE_PLATE
- [x] DARK_OAK_PRESSURE_PLATE
- [x] NAUTILUS_SHELL
- [x] PHANTOM_MEMBRANE
- [x] TUBE_CORAL
- [x] BRAIN_CORAL
- [x] BUBBLE_CORAL
- [x] FIRE_CORAL
- [x] HORN_CORAL
- [x] TUBE_CORAL_FAN
- [x] BRAIN_CORAL_FAN
- [x] BUBBLE_CORAL_FAN
- [x] FIRE_CORAL_FAN
- [x] HORN_CORAL_FAN
- [x] DEAD_TUBE_CORAL_FAN
- [x] DEAD_BRAIN_CORAL_FAN
- [x] DEAD_BUBBLE_CORAL_FAN
- [x] DEAD_FIRE_CORAL_FAN
- [x] DEAD_HORN_CORAL_FAN
- [x] TUBE_CORAL_BLOCK
- [x] BRAIN_CORAL_BLOCK
- [x] BUBBLE_CORAL_BLOCK
- [x] FIRE_CORAL_BLOCK
- [x] HORN_CORAL_BLOCK
- [x] DEAD_TUBE_CORAL_BLOCK
- [x] DEAD_BRAIN_CORAL_BLOCK
- [x] DEAD_BUBBLE_CORAL_BLOCK
- [x] DEAD_FIRE_CORAL_BLOCK
- [x] DEAD_HORN_CORAL_BLOCK
- [x] SEA_PICKLE
- [x] OAK_WOOD
- [x] SPRUCE_WOOD
- [x] BIRCH_WOOD
- [x] JUNGLE_WOOD
- [x] ACACIA_WOOD
- [x] DARK_OAK_WOOD
- [x] STRIPPED_OAK_WOOD
- [x] STRIPPED_SPRUCE_WOOD
- [x] STRIPPED_BIRCH_WOOD
- [x] STRIPPED_JUNGLE_WOOD
- [x] STRIPPED_ACACIA_WOOD
- [x] STRIPPED_DARK_OAK_WOOD
- [x] STRIPPED_OAK_LOG
- [x] STRIPPED_SPRUCE_LOG
- [x] STRIPPED_BIRCH_LOG
- [x] STRIPPED_JUNGLE_LOG
- [x] STRIPPED_ACACIA_LOG
- [x] STRIPPED_DARK_OAK_LOG
- [x] SPRUCE_BUTTON
- [x] JUNGLE_BUTTON
- [x] BIRCH_BUTTON
- [x] ACACIA_BUTTON
- [x] DARK_OAK_BUTTON
- [x] SPRUCE_TRAPDOOR
- [x] BIRCH_TRAPDOOR
- [x] JUNGLE_TRAPDOOR
- [x] ACACIA_TRAPDOOR
- [x] DARK_OAK_TRAPDOOR
- [x] DEBUG_STICK

### Potions

- [x] SLOW_FALLING POTION
- [x] SLOW_FALLING POTION EXTENDED
- [x] SLOW_FALLING LINGERING POTION
- [x] SLOW_FALLING LINGERING POTION EXTENDED
- [x] SLOW_FALLING SPLASH POTION
- [x] SLOW_FALLING SPLASH POTION EXTENDED
- [x] SLOW_FALLING TIPPED ARROW
- [x] SLOW_FALLING TIPPED ARROW EXTENDED

### I added
- [x] CONDUIT
- [x] PRISMARINE_SLAB
- [x] PRISMARINE_STAIRS
- [x] PRISMARINE_BRICK_SLAB
- [x] PRISMARINE_BRICK_STAIRS
- [x] DARK_PRISMARINE_SLAB
- [x] DARK_PRISMARINE_STAIRS
- [x] SEAGRASS
- [x] TALL_SEAGRASS
- [x] SHULKER_BOX
- [x] MUSHROOM_STEM
- [x] SCUTE
- [x] COD_SPAWN_EGG
- [x] DOLPHIN_SPAWN_EGG
- [x] DROWNED_SPAWN_EGG
- [x] PHANTOM_SPAWN_EGG
- [x] PUFFERFISH_SPAWN_EGG
- [x] SALMON_SPAWN_EGG
- [x] TROPICAL_FISH_SPAWN_EGG
- [x] TURTLE_SPAWN_EGG
- [x] TURTLE_MASTER POTION
- [x] TURTLE_MASTER POTION EXTENDED
- [x] TURTLE_MASTER POTION UPGRADED
- [x] TURTLE_MASTER LINGERING POTION
- [x] TURTLE_MASTER LINGERING POTION EXTENDED
- [x] TURTLE_MASTER LINGERING POTION UPGRADED
- [x] TURTLE_MASTER SPLASH POTION
- [x] TURTLE_MASTER SPLASH POTION EXTENDED
- [x] TURTLE_MASTER SPLASH POTION UPGRADED
- [x] TURTLE_MASTER TIPPED ARROW
- [x] TURTLE_MASTER TIPPED ARROW EXTENDED
- [x] TURTLE_MASTER TIPPED ARROW UPGRADED

dscalzi committed:
Removed tabs from items.json.

dscalzi committed:
Fix some alias issues.

dscalzi reviewed (changes requested):
Your additions are good, however I want to add some consistency to the old wood aliases. I'm thinking of removing the strange aliases for pine (such as dwood) and birch (lightwood), as it becomes way too much to manage.

Plus, some items have been renamed in 1.13 so the aliases should be updated accordingly.

Feel free to have a go at this, but I will also be pushing some changes.

dscalzi committed:
Remove dark alias for spruce items.

dscalzi committed:
Remove 'light' and 'white' aliases for birch items.

dscalzi committed:
Further improvements.

Fixed oak item aliases.
Removed forest alias for jungle items.
Added aliases for iron nugget.
Removed {x}dooritem entries.

dscalzi committed:
Rename Stone Brick Monster Egg to 1.13 equivalent.
Renamed to Infested Stone Brick.

dscalzi committed:
Added prismarine stairs + slabs & cleaned stairs aliases.

dscalzi committed:
Remove double slabs as they do not exist in 1.13.

dscalzi committed:
Smooth Sandstone renamed to Cut Sandstone. Also removed the 'smooth' alias for sandstone and stone as there are now blocks with those names.

dscalzi committed:
Cleanup wood aliases, remove pine for spruce.

dscalzi committed:
Added Turtle Master potion.

Also fixed luck potion issue.

dscalzi reviewed (approved):
Lgtm, if anything is missing submit another PR or open an issue 👍
